src/main/scala/com/libertyglobal/odh/kafkatool/config/TopicSettings.scala

```diff
@@ -32,6 +33,7 @@ object TopicSettings {
     TopicSettings(
       config.as[Int]("rf"),
       config.as[Int]("partitions"),
+      if (config.hasPath("access-control-entries")) config.as[Seq[TopicAclEntry]]("access-control-entries") else Seq.empty,
```

Comment:
config.getAs

Reply:
I tried what you suggested yesterday and that, nothing shows up nor seems to work

Reply:
TopicSettingsOverrides.scala:46